### PR TITLE
Update publish-release-notes.md

### PR DESCRIPTION
**Pull request by icarusdust**

### CODE REVIEW

**Comment by icarusdust** [comment]
Fixes #432

File: content/yaml-publishing/publish-release-notes.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Note that release notes are uploaded in the [post-processing step](/yaml-publish
 
 There are three supported options to set up release notes:
 
-1. Create a `release_notes.txt` file and add it to your project working directory, which is either the repository root directory or the `working_directory` specified in the root of your workflow configuration. Codemagic will fetch the content of that file and publish it with the build.
+1. Create a `release_notes.txt` or `release_notes.json` file for Play Store and `release_notes.json` file for App Store Connect, then add it to your project working directory, which is either the repository root directory or the `working_directory` specified in the root of your workflow configuration. Codemagic will fetch the content of that file and publish it with the build.
```
Comment (nevercode-kevin) [inline content/yaml-publishing/publish-release-notes.md:23]:
```suggestion
1. Create a `release_notes.txt` or `release_notes.json` file for Play Store and a `release_notes.json` file for App Store Connect. If your workflow is publishing to both the Play Store and the App Store then it is recommended that you use JSON.  You should then add it to your project working directory, which is either the repository root directory or the `working_directory` specified in the root of your workflow configuration. Codemagic will fetch the content of that file and publish it with the build.
```

**Review by nevercode-kevin — CHANGES_REQUESTED** [review]

File: content/yaml-publishing/publish-release-notes.md (hunk 1 above)
Comment (icarusdust) [reply content/yaml-publishing/publish-release-notes.md]: changes made!

**Review by nevercode-kevin — APPROVED** [review]
LGTM

**Pull request merged**